cpu/src/lib.rs

```diff
@@ -31,8 +31,9 @@ pub(crate) mod native_cpuid {
     }
 
     #[allow(unreachable_code)]
+    #[allow(unused_variables)]
     pub fn cpuid_count(a: u32, c: u32) -> CpuIdResult {
-        #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+        #[cfg(all(target_arch = "x86", target_arch = "x86_64", not(target_env = "sgx")))]
```

Comment:
```suggestion
        #[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), not(target_env = "sgx")))]
```
This way it will still work on 32bit architecture

Reply:
btw, now the two cfg's below doesn't need the sgx cfg guards